src/services/find-available-port.service.js

```diff
@@ -12,13 +12,20 @@
  * hard of a problem!
```

Comment:
This comment is pretty out-of-date. Could I trouble you to change it to:

```
/**
 * Find a clear port to run a server on.
 * 
 * NOTE: Initially, we tried to copy create-react-app's approach, using the
 * `detect-port-alt` NPM package. For some reason, maybe involving electron,
 * that module didn't work; it would create a test server, but they'd hang;
 * no errors called, but no listeners called either.
 * 
 * Instead, we're using platform-specific OS tools: 
 * - `lsof` on Mac/Linux
 * - `netstat` on Windows
 */
```

Reply:
OK, I modified this comment.